pkg/setup/build_crd.go

```diff
@@ -34,7 +34,8 @@ func buildBlockDeviceCRD() (*apiext.CustomResourceDefinition, error) {
 		WithPlural(apis.BlockDeviceResourcePlural).
 		WithShortNames([]string{apis.BlockDeviceResourceShort}).
 		WithPrinterColumns("NodeName", "string", ".spec.nodeAttributes.nodeName").
-		WithPriorityPrinterColumns("Path", "string", ".spec.path", 1).
+		WithPriorityPrinterColumns("Path", "string", ".spec.path", ".spec.FileSystem.Type", 1).
+		WithPrinterColumns("FSType", "string", ".spec.Filesystem.Type").
```

Comment:
The column is `.spec.fileSystem.fsType`

Reply:
The json path can be found here. https://github.com/openebs/node-disk-manager/blob/7a73f6b87cb0a4706ac4a9c89d8212aa4e228dbd/pkg/apis/openebs/v1alpha1/blockdevice_types.go#L164

Reply:
Should it be .spec.filesytem.fsType then?
https://github.com/openebs/node-disk-manager/blob/7a73f6b87cb0a4706ac4a9c89d8212aa4e228dbd/pkg/apis/openebs/v1alpha1/blockdevice_types.go#L63